docs/content/index.md

```diff
@@ -2,60 +2,153 @@
 layout: doc_page
 ---
 
-# About Druid
+Introduction to Druid
+---------------------
 
-Druid is an open-source analytics data store designed for OLAP queries on timeseries data (trillions of events, petabytes of data). Druid provides cost-effective and always-on real-time data ingestion, arbitrary data exploration, and fast data aggregation.
+Druid is an open source data store designed for [OLAP](http://en.wikipedia.org/wiki/Online_analytical_processing) queries on timeseries data.
```

Comment:
I think two pages makes sense.

Splash page:
- One sentence description of druid
- List of key attributes (designed for analytics, low latency, highly available, scalable, multi-tenant, whatever else. Probably 4–8 things)
- Links to next steps. Possibly three options, choose-your-own-adventure style: tutorial, white paper, the first page of the docs

First doc page:
- Everything else you currently have on index.md